src/Distribution/Nix/Fetch.hs

```diff
@@ -225,9 +226,11 @@ prefetch _ fetch@(GitHub {..}) = do
 prefetch _ fetch@(GitLab {..}) = do
   let
     args = ["--name", T.unpack name, "--unpack", T.unpack url]
-    url = "https://gitlab.com/" <> owner <> "/" <> repo
-          <> "/repository/archive.tar.gz?ref=" <> rev
-    name = repo <> "-" <> rev <> "-src"
+    slug = encodeText $ owner <> "/" <> repo
```

Comment:
Is `nix-prefetch-url` going to encode the URL as well? Will we have issues with double encoding here?

Reply:
I don't think `nix-prefetch-url` will encode the URL. This is the way it's done in Nixpkgs, so at least we are being consistent: if I'm wrong, and there is a double-encoding bug, I think Nixpkgs will have the same bug.